handlers/handlers.go

```diff
@@ -25,29 +39,157 @@ func setStatusCode(req *http.Request, w http.ResponseWriter, err error) {
 	w.WriteHeader(status)
 }
 
-// HelloWorld Handler
-func HelloWorld(cfg config.Config) http.HandlerFunc {
+// getCookiePreferencePage talks to the renderer to get the cookie preference page
+func getCookiePreferencePage(ctx context.Context, w http.ResponseWriter, req *http.Request, rendC RenderClient, cp cookies.Policy) error {
```

Comment:
Is ctx needed? As the caller to function passes a http.Request containing context

I would do the following instead:

```
getCookiePreferencePage(ctx context.Context, w http.ResponseWriter, req *http.Request, rendC RenderClient, cp cookies.Policy) error {
    ctx := req.Context()
    ...
}
```